lib/open.js

```diff
@@ -1,52 +1,44 @@
 var exec = require('child_process').exec;
 var number = require('./utils').number;
-var quote = require('./utils').quote;
+var shell = require('./editors/common/shell');
 var extractFilename = require('./utils').extractFilename;
-var append = require('./utils').append;
+var assign = require('./utils').assign;
+var terminal = require('./editors/common/terminal');
```

Comment:
Hm, I looked in raw file – it's ok.
There is a bug on github or in one of plugin I' using:
![screen shot 2017-04-04 at 01 46 38](https://cloud.githubusercontent.com/assets/270491/24634841/a9256498-18d8-11e7-9ad3-cd476609744d.png)